buildconfig/manylinux-build/docker_base/Dockerfile-aarch64

```diff
@@ -0,0 +1,75 @@
+ARG BASE_IMAGE=manylinux2014_aarch64
```

Comment:
Why does this need a separate Dockerfile? The base image can be passed in as an argument, so I don't see what the need is for a separate file.